packages/compass-import-export/src/modules/import.ts

```diff
@@ -252,17 +253,27 @@ export const startImport = () => {
 
     const ignoreBlanks = ignoreBlanks_ && fileType === FILE_TYPES.CSV;
     const fileSize = fileStats?.size || 0;
-
     const fields: Record<string, CSVParsableFieldType> = {};
     for (const [name, type] of transform) {
       if (exclude.includes(name)) {
         continue;
       }
       fields[name] = type;
     }
-
     const input = fs.createReadStream(fileName, 'utf8');
 
+    const errors: ErrorJSON[] = [];
+
+    let errorLogFilePath;
+    try {
+      errorLogFilePath = await getErrorLogPath(fileName);
+    } catch (err: any) {
+      errors.push(err as Error);
+    }
+    const errorLogWriteStream = errorLogFilePath
+      ? fs.createWriteStream(errorLogFilePath)
```

Comment:
Are we sure opening a write stream will always work? Should this be guarded by try catch or do we want to hard fail if for some reason we can't write logs to the file?

Reply:
Ah thanks, should be a few lines up in the try catch. Updated.